### ISSUE TITLE
Question:  The difference in gradients between interpolate and antialias

### ISSUE THREAD

**Issue by yuedajiong** [NVlabs/nvdiffrast#212]
1. After interpolation, good rendering results have been achieved, and we can directly use loss.backward() to obtain gradients. However, the gradients obtained  are less accurate than backward after antialiasing. Why is this the case? For rendering speed, I would like to only perform rasterization and interpolation in the early stages of fitting. However, the gradients (including pos and col) are not very accurate. This means I must perform the antialias operation throughout the entire fitting process.

2. Can the gradients (including pos and col) obtained after antialiasing be used to guide the growth and pruning of vertices and faces?

Thanks ！
